src/database/seeders/20200619191649-add-escolas.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('Escola', [
       { tipo_escola: 'Não registrada' },
-      { tipo_escola: 'Pública e privada, tendo ficado mais tempo em escola pública' },
+      { tipo_escola: 'Pública e privada, tendo ficado mais tempo em escola pblica' },
```

Comment:
Qual o motivo de colocar sem acento? Além disso, no início da frase tem "Pública" com acento. Se por algum motivo não é interessante colocar texto acentuado, então mantém a letra sem o acento, ie. ao invés de pblica, seria publica.

Reply:
É porque no início eu estava colocando na ortografia correta, mas quando o Sr. me mandou os .csv eu alterei para deixar igual aos .csv. Mas irei voltar a forma que estava anteriormente.